spec/support/page_objects/publish/school_new.rb

```diff
@@ -8,12 +8,12 @@ class SchoolNew < PageObjects::Base
       set_url '/publish/organisations/{provider_code}/{recruitment_cycle_year}/schools/new'
       element :error_summary, '.govuk-error-summary'
 
-      section :school_form, Sections::SchoolForm, '.school-form'
+      # section :school_form, Sections::SchoolForm, '.school-form'
 
-      element :name_field, '#publish-school-form-location-name-field'
-      element :address1_field, '#publish-school-form-address1-field'
-      element :address3_field, '#publish-school-form-address3-field'
-      element :postcode_field, '#publish-school-form-postcode-field'
+      element :name_field, '#support-school-form-location-name-field'
```

Comment:
Not sure if this should reference the support one. There's an existing one that's provider specific

Reply:
Again, the support form has the store.

Reply:
But let me check this

Reply:
The id comes from the Support::SchoolForm